jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/BreadcrumbTests.java

```diff
@@ -0,0 +1,139 @@
+package io.github.epam.bootstrap.tests.common;
+
+import com.epam.jdi.light.driver.WebDriverFactory;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.WebPage;
+import io.github.epam.TestsInit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+
+import static com.epam.jdi.light.common.TextTypes.TEXT;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.breadcrumb;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * Created by Olga Ivanova on 16.09.2019
+ * Email: olga_ivanova@epam.com
+ */
+
+public class BreadcrumbTests extends TestsInit {
+    private static final String HOME = "Home";
+    private static final String HTML5 = "HTML 5";
+    private static final String BOOTSTRAP = "Bootstrap";
+    private static final String[] ITEMS_VALUES = new String[]{HOME, HTML5, BOOTSTRAP};
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void getTextTest() {
+        breadcrumb.items.has().size(ITEMS_VALUES.length);
+        breadcrumb.items.assertThat().values(TEXT, hasItems(ITEMS_VALUES));
+    }
+
+    @Test
+    public void getValueTest() {
+        breadcrumb.items.has().size(ITEMS_VALUES.length);
+        assertThat(breadcrumb.items.values().toArray(), is(ITEMS_VALUES));
+    }
+
+    @Test
+    public void getFirstItemTest() {
+        breadcrumb.items.first().has().value(HOME);
+        breadcrumb.items.first().is().text(HOME);
+    }
+
+    @Test
+    public void getCurrectItemTest() {
+        breadcrumb.items.last().has().value(BOOTSTRAP);
+        breadcrumb.items.last().has().text(BOOTSTRAP);
+    }
+
+    @Test
+    public void clickCurrectItemTest(){
+        breadcrumb.items.last().click();
+
+        ArrayList<String> tabs = new ArrayList<>(WebDriverFactory.getDriver().getWindowHandles());
+        assertThat(tabs.size(), is(1));
+    }
+
+    @Test
+    public void clickByNameTest() {
+        breadcrumb.items.get(HTML5).click();
+
+        ArrayList<String> tabs = new ArrayList<>(WebDriverFactory.getDriver().getWindowHandles());
+        WebDriver driver = WebDriverFactory.getDriver();
+        driver.switchTo().window(tabs.get(1));
```

Comment:
I find out that we already have WindowsManager so we should use it instead of direct calls to driver, be careful WindowsManager uses 1 as a first index instead of 0 so possibly you also will need to update index values used in tests

Reply:
done